src/App.tsx

```diff
@@ -23,181 +124,123 @@ export const App: React.FC = () => {
             </div>
-          </div>
-        </section>
+          </section>
+        ))}
 
         {/* Hide the footer if there are no todos */}
-        <footer className="todoapp__footer" data-cy="Footer">
-          <span className="todo-count" data-cy="TodosCounter">
-            3 items left
-          </span>
-
-          {/* Active link should have the 'selected' class */}
-          <nav className="filter" data-cy="Filter">
-            <a
-              href="#/"
-              className="filter__link selected"
-              data-cy="FilterLinkAll"
-            >
-              All
-            </a>
 
-            <a
-              href="#/active"
-              className="filter__link"
-              data-cy="FilterLinkActive"
-            >
-              Active
-            </a>
+        {loadContent.length !== 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {`${loadContent.filter(item => !item.completed).length} items left`}
+            </span>
 
-            <a
-              href="#/completed"
-              className="filter__link"
-              data-cy="FilterLinkCompleted"
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              {filteredButtons.map(button => (
+                <a
+                  href={button.href}
+                  className={classNames(`${button.className}`, {
+                    selected: filteredBy === button.name,
+                  })}
+                  data-cy={button.dataCy}
+                  key={button.key}
+                  onClick={handleFilterButtons}
+                >
+                  {button.name}
+                </a>
+              ))}
+            </nav>
+
+            {/* this button should be disabled if there are no completed todos */}
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              data-cy="ClearCompletedButton"
             >
-              Completed
-            </a>
-          </nav>
-
-          {/* this button should be disabled if there are no completed todos */}
-          <button
-            type="button"
-            className="todoapp__clear-completed"
-            data-cy="ClearCompletedButton"
-          >
-            Clear completed
-          </button>
-        </footer>
+              Clear completed
+            </button>
+          </footer>
+        )}
       </div>
 
       {/* DON'T use conditional rendering to hide the notification */}
       {/* Add the 'hidden' class to hide the message smoothly */}
       <div
         data-cy="ErrorNotification"
-        className="notification is-danger is-light has-text-weight-normal"
+        className={classNames(
+          'notification is-danger is-light has-text-weight-normal',
+          {
+            hidden: errorMessege.length === 0,
```

Comment:
```suggestion
            hidden: !!errorMessage
```
empty string it is falsy value. no need to compare length.